requirements.txt

```diff
@@ -2,5 +2,5 @@ cwl-upgrader >= 1.2.3
 packaging
 rdflib
 requests
-schema-salad >= 8.8, < 9
+schema-salad @ git+https://github.com/common-workflow-language/schema_salad.git
```

Comment:
Is this still needed?

Reply:
No, but we need `>= 8.8.20241206093842` for the proper detection of namespaced classes

Reply:
Oh no that's not true. This is true just during generation, not at runtime.